cirq/google/engine/client/quantum_v1alpha1/gapic/quantum_engine_service_client.py

```diff
@@ -242,12 +242,15 @@ def create_quantum_program(
                     to a retryable error and retry attempts failed.
             ValueError: If the parameters are invalid.
         """
-        # Wrap the transport method to add retry and timeout logic.
+        # Wrap the transport method to add retry, timeout, and compression logic.
         if 'create_quantum_program' not in self._inner_api_calls:
             self._inner_api_calls[
                 'create_quantum_program'
             ] = google.api_core.gapic_v1.method.wrap_method(
-                self.transport.create_quantum_program,
+                functools.partial(
+                    self.transport.create_quantum_program,
+                    compression=grpc.Compression.Gzip,
```

Comment:
The PR description points to a Java example but doesn't really reference how it maps to this. Mind documenting somewhere where this `compression` parameter is defined? IIUC, it comes from the [`Channel`](https://github.com/grpc/grpc/blob/86f93801e5d452cff40d6958d21a8eb211da0654/src/python/grpcio/grpc/_channel.py#L1345) class, but that was a real pain to track down.